package.json

```diff
@@ -4,23 +4,19 @@
   "description": "Mozilla Front End Infra Shareable React Components",
   "repository": "mozilla-frontend-infra/components",
   "license": "MPL-2.0",
-  "main": "build/index.js",
+  "main": "./index.js",
```

Comment:
Doesn't this still build to the `build` directory?

Reply:
Yes, but the `package.json` will be copied to the `build` directory. Inside `.neutrinorc.js` I added:

```js
neutrino.on('build', () => {
  ['package.json', 'logo.png', 'LICENSE', 'README.md', 'AUTHORS'].map(file => {
    fs.copyFileSync(file, join(__dirname, `build/${file}`));
  })
});
```